apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/MapLikeToContainInOrderOnlyCreatorSamples.kt

```diff
@@ -6,6 +6,30 @@ import kotlin.test.Test
 
 class MapLikeToContainInOrderOnlyCreatorSamples {
 
+    @Test
+    fun entry() {
+        expect(mapOf(1 to "a")).toContain.inOrder.only.entry(1 to "a")
+
+        fails {
+            expect(mapOf(1 to "a", 2 to "b")).toContain.inOrder.only.entry(
+                1 to "a", // fails because subject does not have the same order
```

Comment:
order is correct bu there is an additional entry:
```suggestion
                1 to "a",  // fails because subject has additional entries
```